lib/app.js

```diff
@@ -205,11 +197,55 @@ module.exports = function(tilelive, options) {
             headers["content-encoding"] = headers["content-encoding"] || "gzip";
           }
 
-          res.set(populateHeaders(headers, params, { 200: true }));
-          return res.send(data);
+          return callback(null, data, populateHeaders(headers, params, { 200: true }));
         });
       });
     });
+  };
+
+  app.get(tilePattern, function(req, res, next) {
+    return getTile(req.params.z | 0, req.params.x | 0, req.params.y | 0,
+        (req.params.scale || "@1x").slice(1, 2), req.params.format,
+        function(status, data, headers) {
+          if (status && Number(status) == NaN) {
+            return next(status);
+          }
+          res.set(headers);
+          res.status(status || 200);
+          return res.send(data);
+        }, res, next);
+  });
+
+  var floatPattern = "[+-]?(?:\\d+|\\d+\.?\\d+)";
+  var staticPattern =
+    "/static/:lon(" + floatPattern + "),:lat(" + floatPattern + "),:z(\\d+)/" +
+    ":width(\\d+)x:height(\\d+):scale(@[123]x)?\.:format([\\w\\.]+)";
+
+  app.get(staticPattern, function(req, res, next) {
+    var scale = (req.params.scale || "@1x").slice(1, 2) | 0;
```

Comment:
Please merge the `var` definitions to match style elsewhere.

Also consider extracting the `scale` determination as `getScale(scale)`, since this appears more than once now (and isn't totally straightforward).